manifests/bucketeer/charts/event-persister-evaluation-events-dwh/templates/deployment.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "event-persister-evaluation-events-dwh.fullname" . }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ template "event-persister-evaluation-events-dwh.fullname" . }}
+    chart: {{ template "event-persister-evaluation-events-dwh.chart" . }}
+    release: {{ template "event-persister-evaluation-events-dwh.fullname" . }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "event-persister-evaluation-events-dwh.name" . }}
+      release: {{ template "event-persister-evaluation-events-dwh.fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "event-persister-evaluation-events-dwh.name" . }}
+        release: {{ template "event-persister-evaluation-events-dwh.fullname" . }}
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/envoy-configmap.yaml") . | sha256sum }}
+    spec:
+      {{- with .Values.global.image.imagePullSecrets }}
+      imagePullSecrets: {{- toYaml . | nindent 8 }}
+      {{- end }}
+      affinity:
+{{ toYaml .Values.affinity | indent 8 }}
+      nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 8 }}
+      volumes:
+        - name: envoy-config
+          configMap:
+            name: {{ template "event-persister-evaluation-events-dwh.fullname" . }}-envoy-config
+        - name: service-cert-secret
+          secret:
+            secretName: {{ template "service-cert-secret" . }}
+        - name: service-token-secret
+          secret:
+            secretName: {{ template "service-token-secret" . }}
```

Comment:
please insert `serviceAccountName: {{ template "event-persister-evaluation-events-dwh.name" . }}`

Reply:
Sorry for the small detail, can you insert `serviceAccountName` between `volumes` and `containers` like other files for consistency?

eg. https://github.com/bucketeer-io/bucketeer/blob/1d86fe83f19333b0c106ecd967fb4f7cf280f291/manifests/bucketeer/charts/event-persister-evaluation-events-kafka/templates/deployment.yaml#L31-L42